docs/source/workflow/director_based_workflow.rst

```diff
@@ -0,0 +1,362 @@
+.. # Copyright (C) 2020 Intel Corporation
+.. # Licensed subject to the terms of the separately executed evaluation license agreement between Intel Corporation and you.
+
+.. _director_workflow:
+
+************************
+Director-based workflow
+************************
+
+.. toctree::
+    :maxdepth: 2
+ 
+    establishing_federation_director_
+    interactive_api_
+
+
+.. _establishing_federation_director:
+
+Establishing a long-living Federation with Director
+#######################################
+
+1. Install |productName| 
+==================
+
+Make sure that you installed |productName| in your virtual Python environment.
+If not, use the instruction :ref:`install_initial_steps`.
+
+2. Implement Shard Descriptors
+==================
+
+Then the data owners need to implement `Shard Descriptors` Python classes. 
+
+|productName| framework provides a ‘Shard descriptor’ interface that should be described on every collaborator node 
+to provide a unified data interface for FL experiments. Abstract “Shard descriptor” should be subclassed and 
+all its methods should be implemented to describe the way data samples and labels will be loaded from disk 
+during training. Shard descriptor is a subscriptable object that implements :code:`__getitem__()` and :code:`len()` methods 
+as well as several additional methods to access ‘sample shape’, ‘target shape’, and ‘shard description’ text 
+that may be used to identify participants during experiment definition and execution.
+
+3. (Optional) Create certificates using Step-CA 
+==================
+
+All communications inside a Federation may be encrypted with mTLS. User may adapt certificates provided by their organization 
+or utilize :ref:`PKI <semi_automatic_certification>` provided by |productName|.
+
+4. Start Director
+==================
+
+Director is a central component in the Federation. It should be started on a node with at least one open port. 
```

Comment:
@igor-davidyuk this should be two open ports, right? One for the director and a second for the aggregator it spawns

Reply:
correct, fixed